PerformanceTests/JetStream2/JetStreamDriver.js

```diff
@@ -233,6 +244,16 @@ class Driver {
             }
 
             benchmark.updateUIAfterRun();
+
+            if (isInBrowser) {
+                let cache = window.blobDataCache;
```

Comment:
Let's make `this.blobDataCache`. since `Driver` is alive, we do not need to create a window's variable.

Reply:
And let's have `this.blobDataCache = {};` in the constructor of this `Driver`.

Reply:
I'll put it in the `Driver` but reference it using `JetStream.blobDataCache`.  `JetStream` is the global singleton instance of `Driver`.  Some of the places below are in the `Benchmark` class and not `Driver` itself.  I'll just use `JetStream.blobDataCache` everywhere for consistency.

Ditto for `JetStream.loadCache`.